src/System.Net.Http.Json/Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <AssemblyVersion>3.2.0.0</AssemblyVersion>
+    <PackageVersion>3.2.0</PackageVersion>
+
+    <!-- Setting Suppress Final Package Version and IsShippingAssembly properties in order to force the release build to 
+    continue emitting pre-release labels as we won't want to ship stable on this release yet. -->
+    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
```

Comment:
We don't need to override this label, we already set it to preview3 as we want to align with aspnetcore branding.

Reply:
Will fix, this was set for release/3.1 PR since the label there was servicing.